Candidate: CVE-2022-42969
PublicDate: 2022-10-27
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-42969
 https://github.com/pytest-dev/py/issues/287
 https://github.com/pytest-dev/py/issues/287#issuecomment-1283784766
Description:
 (The py library through 1.11.0 for Python allows remote attackers to co
 ...)
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]

Patches_py:
upstream_py: needs-triage
trusty_py: ignored (out of standard support)
xenial_py: ignored (out of standard support)
bionic_py: DNE
focal_py: DNE
jammy_py: DNE
kinetic_py: DNE

Patches_python-py:
upstream_python-py: needs-triage
trusty_python-py: ignored (out of standard support)
xenial_python-py: ignored (out of standard support)
bionic_python-py: needs-triage
focal_python-py: needs-triage
jammy_python-py: needs-triage
kinetic_python-py: needs-triage
